<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ragdoll Tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>RigidBody Parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Mass</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="massLE">
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Restitution</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="restitutionLE">
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Friction</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="frictionLE">
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Constraint</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Friction Torque</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="maxFrictionTorqueLE"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
